index.html

```diff
@@ -120,5 +120,40 @@ <h5>Next level iceland blue bottle fingerstache:</h5>
         </ul>
       </section>
     </section>
+    <section class="contact">
+      <h2>contact</h2>
+      <form>
+        <label for="name">name</label>
+        <input type="text" id="name">
+        <label for="email">e-mail</label>
+        <input type="email" id="email">
+        <label for="topic">topic</label>
+        <input type="text" id="topic">
```

Comment:
I know it doesn't look like it, but if someone is to write them an email probably could use a bigger space for that. I would probably use `<textarea>` here.

Reply:
I had this thought too, changing this. :)